### ISSUE TITLE
[SCREENREADER] HCA - dl, dt, and dd are not structured correctly

### ISSUE THREAD

**Issue by bkjohnson** [department-of-veterans-affairs/va.gov-team#13999]
##  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This is a follow-up for #13647

In the `hca` app, Cypress is reporting some [axe errors](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/13647-hca/1/pipeline):

```
[2020-09-23T22:16:12.131Z]   Running:  [90mapplications/hca/tests/hca.cypress.spec.js[39m                                    [90m(27 of 51)[39m
[2020-09-23T22:16:13.053Z] [info] [webpackbar] Compiling Webpack
[2020-09-23T22:16:18.286Z] [success] [webpackbar] Webpack: Compiled successfully in 5.03s
[2020-09-23T22:16:22.441Z] 
[2020-09-23T22:16:22.441Z] [0m[0m
[2020-09-23T22:16:22.441Z] [0m  1010ez Health Care Application form[0m
[2020-09-23T22:16:22.441Z] [0m    minimal-test[0m
[2020-09-23T22:16:34.591Z] 2 accessibility violations were detected
[2020-09-23T22:16:34.591Z] ┌─────────┬───────────────────┬───────────┬──────────────────────────────────────────────────────────┬───────┐
[2020-09-23T22:16:34.591Z] │ (index) │        id         │  impact   │                       description                        │ nodes │
[2020-09-23T22:16:34.591Z] ├─────────┼───────────────────┼───────────┼──────────────────────────────────────────────────────────┼───────┤
[2020-09-23T22:16:34.591Z] │    0    │ 'definition-list' │ 'serious' │     'Ensures <dl> elements are structured correctly'     │   1   │
[2020-09-23T22:16:34.591Z] │    1    │     'dlitem'      │ 'serious' │ 'Ensures <dt> and <dd> elements are contained by a <dl>' │   1   │
[2020-09-23T22:16:34.591Z] └─────────┴───────────────────┴───────────┴──────────────────────────────────────────────────────────┴───────┘
[2020-09-23T22:16:34.591Z]     [31m  1) fills the form[0m
[2020-09-23T22:16:34.591Z] [0m    maximal-test[0m
[2020-09-23T22:17:42.257Z]     [31m  2) fills the form[0m
[2020-09-23T22:17:42.257Z] [0m    foreign-address-test[0m
[2020-09-23T22:18:38.389Z]     [31m  3) fills the form[0m
[2020-09-23T22:18:38.389Z] 
[2020-09-23T22:18:38.389Z] 
[2020-09-23T22:18:38.389Z] [92m [0m[32m 0 passing[0m[90m (2m)[0m
[2020-09-23T22:18:38.389Z] [31m  3 failing[0m
[2020-09-23T22:18:38.389Z] 
[2020-09-23T22:18:38.389Z] [0m  1) 1010ez Health Care Application form
[2020-09-23T22:18:38.389Z]        minimal-test
[2020-09-23T22:18:38.389Z]          fills the form:
[2020-09-23T22:18:38.389Z] [0m[31m     AssertionError: 2 accessibility violations were detected: expected 2 to equal 0[0m[90m
[2020-09-23T22:18:38.389Z]       at Context.eval (http://vets-website:3001/__cypress/tests?p=src/platform/testing/e2e/cypress/support/index.js:24606:443712)
```

A PR can be made from this branch: https://github.com/department-of-veterans-affairs/vets-website/compare/13647-hca

## Acceptance Criteria

- [x] Fix dl, dt, and dd structure
- [ ] The temporary exception for issue 13647 is removed
- [ ] The hca app passes all Cypress axe checks

## Possible Fixes (optional)

This may be an issue with how the forms library is rendering what the hca app is telling it to render. More investigation will need to be done to find the root cause.

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by jenstrickland** [comment]
@ala-yna 👋  Hey there, do you still have this on your team's radar?

**Comment by ala-yna** [comment]
Yep, we haven't had the capacity to start development on the HCA work yet. But still on the radar!

**Comment by LinnJS** [comment]
After running the e2e test found that the error is happening on the review page. Found that the errors highlighted in red are related to the dotted outline around that particular UI. Going to dive deeper and figure out how the data might be causing the UI to render this way.

![Screen Shot 2020-12-03 at 4.48.39 PM.png](https://images.zenhubusercontent.com/5dcf313b98e0b800015f2353/e2074b95-61c4-47fb-a89d-ca2ef67a2709)

**Comment by LinnJS** [comment]
508 Discussion regarding other errors on the form.
https://dsva.slack.com/archives/C8E985R32/p1607469704254900

**Comment by LinnJS** [comment]
This PR was merged and will be in staging by end of day.

**Comment by ala-yna** [comment]
Marking this as closed. @joshkimux, any thoughts?

**Comment by joshkimux** [comment]
@ala-yna no thoughts!

**Comment by bkjohnson** [comment]
It seems like the acceptance criteria wasn't met (removing the temporary exception for #13647). Am I understanding that correctly?

**Comment by bkjohnson** [comment]
Reopening since the issue isn't resolved as I understand it.

**Comment by bkjohnson** [comment]
The exception still exists in `hca` as of today:

![image](https://user-images.githubusercontent.com/2008881/105909777-dcfdda80-5fdc-11eb-91ac-6bb1bfcd4498.png)

**Comment by LinnJS** [comment]
@bkjohnson @ala-yna If you look at the discussion I posted above when I merged the PR for this ticket originally there was many contrast bugs with the VSP testing platform. It does not feel fair that the A/Cs were blocked due to bugs after I spent a lot of time trying to fix it and investigate why...

It seems that it might be fixed now but I feel strongly that another ticket should be spun up due to time spent. I met the objectives to best of my ability at the time.

Here is the link to the discussion once again https://dsva.slack.com/archives/C8E985R32/p1607469704254900

**Comment by bkjohnson** [comment]
Thank you - Looks good!
